cli/tests/e2e/snapshots/test_ci/test_full_run/autofix-buildkite/results.txt

```diff
@@ -44,11 +44,10 @@ Findings:
 Scan environment:
   versions    - semgrep <MASKED> on python <MASKED>
   environment - running in environment buildkite, triggering event is pull_request
-  semgrep.dev - authenticated as org_name
+  server      - https://semgrep.dev
 
 Fetching configuration from semgrep.dev
-Adding ignore patterns configured on semgrep.dev as `--exclude` options: ()
```

Comment:
This line also should never have been in these snapshots. It was only because the ignores were equal to a mocked-value which then evaluated truthy because it wasn't None or [].